### PR TITLE
MICROBA-447 Add demographics to GDPR pipeline

### PR DESCRIPTION
**Pull request by crice100**
PR to add the endpoint in demographics: https://github.com/edx/demographics/pull/44

### CODE REVIEW

File: tubular/scripts/helpers.py [hunk 1]
```diff
@@ -141,16 +141,17 @@ def _setup_lms_api_or_exit(fail_func, fail_code, config):
...
+    Demographics, as well as fetching the learner's record from LMS and validating that
+    it is in a state to work on. Returns the learner dict and their current stage in the
     retirement flow.
     """
     try:
         lms_base_url = config['base_urls']['lms']
         ecommerce_base_url = config['base_urls'].get('ecommerce', None)
         credentials_base_url = config['base_urls'].get('credentials', None)
         segment_base_url = config['base_urls'].get('segment', None)
+        demographics_base_url = config['base_urls'].get('demographics', None)
```
Comment (tuchfarber) [inline tubular/scripts/helpers.py:154]: I'm trying to remember where this config file is located, but we'll need to add the demographics URL in it. I feel like it's somewhere weird, like a Jenkins configuration file or something.

Comment (justinhynes) [reply tubular/scripts/helpers.py:154]: We started to talk about this during DE's office hours last week. I believe Troy said that it is in a yaml file within the `edx-ops` repo. He also mentioned something about "user retirement secure" when we were discussing it (part of `edx-ops`?).

**Comment by justinhynes** [comment]
Disregard this PR. Closing and opening a new PR as this one was not updating as expected.

**Comment by crice100** [comment]
"Real" PR: https://github.com/edx/tubular/pull/433

**Pull request closed without merging**